library/std/src/process.rs

```diff
@@ -1725,6 +1725,18 @@ impl ExitCode {
     /// return the same codes (but will also `eprintln!` the error).
     #[stable(feature = "process_exitcode", since = "1.60.0")]
     pub const FAILURE: ExitCode = ExitCode(imp::ExitCode::FAILURE);
+
+    /// Exit the current process with the given ExitCode.
+    ///
+    /// Note that this has some of the same caveats as [`process::exit`][exit], namely that this
```

Comment:
There's not actually any difference between this and `process::exit` other than the fact that this accepts fewer inputs, right?  Let me know what you think of this change.
```suggestion
    /// Note that this has the same caveats as [`process::exit`][exit], namely that this
```

Also wondering if it would adding a `# Differences from process::exit` section that clarifies that the range of valid inputs is the only difference, and that this API is intended to be more portable.